docker/Dockerfile

```diff
@@ -1,14 +1,22 @@
-FROM alpine:3.14
-LABEL maintainer="Quickwit, Inc. <hello@quickwit.io>"
+FROM alpine:3.14 AS builder
 
 # Install tools
 RUN apk add --no-cache curl
-
-# Install Quickwit
 RUN curl -L https://install.quickwit.io | sh
-RUN mv ./quickwit /usr/local/bin/quickwit
+RUN mv  ./quickwit-*/* ./
```

Comment:
Wonder if we should `sed` the config here to set the listen address to `0.0.0.0`.
@fmassot ?

Reply:
From the feedback on the other PR, we have a docker flavor of the config. We can copy that file too?

Reply:
@guilload I suggested having a default config file suited for docker. 
So in this case we have the right config file from the start. Sed is fine too, I'm ok with both solutions.